.travis.yml

```
@@ -22,23 +27,23 @@ branches:
     - master
 
 before_script:
+  - export JULIA_PROJECT=${HOME}/.julia/environments/v1.0/
   - printenv | sort
   - git show --pretty=fuller -s
   - mkdir -p coverage
   - etc/install_julia_gap.sh
-  - ln -s $TRAVIS_BUILD_DIR/JuliaInterface $GAPROOT/pkg/
-  - ln -s $TRAVIS_BUILD_DIR/JuliaExperimental $GAPROOT/pkg/
-  - ln -s $TRAVIS_BUILD_DIR/LibGAP.jl $GAPROOT/pkg/
+    ## Add the package so we can use it later
+  - julia --project="$JULIA_PROJECT" -e 'import Pkg ; Pkg.add(Pkg.PackageSpec(url="$(ENV["TRAVIS_BUILD_DIR"])"))'
```

Since `JULIA_PROJECT` is explicitly checked by `julia`, I don't think we need to pass it to Julia. Also, we are in the package dir... So how about this instead?
```suggestion
  - julia -e 'import Pkg ; Pkg.add(Pkg.PackageSpec(path="."))'
```